guides/common/modules/proc_cloning-to-rhel9.adoc

```diff
@@ -0,0 +1,60 @@
+[id="cloning-to-rhel9_{context}"]
+= Cloning to the {RHEL}{nbsp}9 Server
```

Comment:
All of that is already in the Admin guide, so IMHO no need to duplicate, just link?